ee-9/common/pom.xml

```diff
@@ -626,6 +626,16 @@
 
         <!-- Misc -->
 
+        <dependency>
+            <groupId>org.antlr</groupId>
+            <artifactId>antlr4</artifactId>
+        </dependency>
```

Comment:
Please add an entry for this to https://github.com/wildfly/wildfly/blob/main/ee-9/common/src/main/resources/license/preview-feature-pack-common-licenses.xml

Reply:
using:
```
          <name>The Antlr4 4.9.1 License</name>
          <url>https://github.com/antlr/antlr4/blob/4.9.1/LICENSE.txt</url>
          <distribution>repo</distribution>
```

Reply:
Thanks Brian, entry added!